qa/tasks/check_counter.py

```diff
@@ -50,6 +52,8 @@ def end(self):
         if cluster_name is None:
             cluster_name = next(iter(self.ctx.managers.keys()))
 
+        active_mgr = json.loads(self.mon_manager.raw_cluster_cmd("mgr", "dump", "--format=json-pretty"))["active_name"]
```

Comment:
This patch is failing here in testing. It's not finding `mon_manager`. I will fix this up.

Reply:
Fixed